pkg/template/mixer/BUILD

```diff
@@ -0,0 +1,26 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_prefix")
+load("@org_pubref_rules_protobuf//go:rules.bzl", "go_proto_library")
+
+go_proto_library(
```

Comment:
OK. I think we will end up revisiting this soon, but it doesn't need to be a blocker now.